docs/java/java-gui.md

```diff
@@ -0,0 +1,27 @@
+# Working with GUI application in Visual Studio Code
+
+You can develop Java GUI applications in Visual Studio Code easily. To achieve that, you need to install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes all the required extensions to develop Java GUI application.
+
+If you run into any issues when using the features below, you can contact us by entering an [issue](https://github.com/microsoft/vscode-java-pack/issues).
```

Comment:
can we also provide some guidance for unmanaged projects? e.g. what are the steps they will need to take